packages/botonic-plugin-flow-builder/src/index.ts

```diff
@@ -179,8 +165,7 @@ export default class BotonicPluginFlowBuilder implements Plugin {
       contents.push(content)
     }
     // TODO: prevent infinite recursive calls
-
-    if (node.follow_up) {
+    if (node.follow_up && !(content instanceof FlowBotAction)) {
```

Comment:
store the condition in a variable which represents clear this case, or create a private/auxiliary function to encapsulate this

Reply:
applied